nuxeo-compound-documents/src/main/resources/OSGI-INF/service-contrib.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.compound.documents.service">
+  <documentation>
+    Compound Documents Manager, delegating logic to the appropriate compound document type.
+  </documentation>
+  <implementation class="org.nuxeo.compound.documents.CompoundDocumentServiceImpl" />
+  <service>
+    <provide interface="org.nuxeo.compound.documents.CompoundDocumentService" />
+  </service>
+  <extension-point name="ignoredFiles">
+    <documentation>
+      Extension point to register the file names to ignore when importing a compound document archive.
+    </documentation>
+    <object class="org.nuxeo.compound.documents.IgnoredFilesDescriptor" />
+  </extension-point>
+  <documentation>
+    This is a sample ignore file. File names listed here are ignored when importing a compound document archive.
```

Comment:
This is not a sample, this is the default ignore list that is used.

Reply:
ok 2 slip...